.github/workflows/prettier.yml

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+# SPDX-FileCopyrightText: 2023 Tristan Partin <tristan@partin.io>
+
+name: Prettier
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - README.md
```

Comment:
```
- **.json
- **.md
- **.ts
- **.yml
```

Reply:
Heh, silly me.

That said, given the `prettier:check` script specifies the patterns, might as well not filter here? Not like PRs are flying in so even if there's a perf hit surely not an issue...

Reply:
That is fine. Can always change it later.